sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -450,7 +450,22 @@ export class ServiceBusManagementClient extends ServiceClient {
   /**
    * Updates properties on the Queue by the given name based on the given options
    * @param queue Options to configure the Queue being updated.
-   * For example, you can configure a queue to support partitions or sessions.
+   *
+   * Update request allows only a subset of properties from QueueDescription to be changed
+   *  https://docs.microsoft.com/en-us/rest/api/servicebus/update-queue
+   *  Updatable properties:
+   *   - defaultMessageTimeToLive
+   *   - lockDuration
+   *   - deadLetteringOnMessageExpiration
+   *   - duplicateDetectionHistoryTimeWindow
+   *   - maxDeliveryCount
+   *
+   * The `queue` param must be fully populated as all of the properties are replaced with the update API.
+   * If a property is not set, the service default value is used.
+   * The suggested flow is:
+   *  - Use `await getQueue(<queue-name>)` and obtain the getResponse.
+   *  - Update the required elements from the list of updatable properties in the `getResponse`.
+   *  - Pass the updated description into this method.
```

Comment:
The below pieces of info should appear before the `@param` description for the queue, otherwise they appear in the wrong place in the editors.
- subset of properties supported
- but we need the complete description
- suggested flow

My recommendation would be:

```
Updates the queue based on the queue description provided. 

All properties on the queue description must be set even though only a subset of them
are actually updatable. Therefore, the suggested flow is  to use `getQueue()` to get the
queue description with all properties set, update as needed and then pass it to `updateQueue()`.
See https://docs.microsoft.com/en-us/rest/api/servicebus/update-queue for more details.

@param queue Object representing the queue with one or more of the below properties updated
- defaultMessageTimeToLive
- lockDuration
- deadLetteringOnMessageExpiration
- duplicateDetectionHistoryTimeWindow
- maxDeliveryCount

Reply:
Updated as suggested